R/arg-homogenise.R

```diff
@@ -0,0 +1,219 @@
+#' Standardise a call against formal arguments.
+#'
+#' This is a slower but more thorough version of
```

Comment:
Can you add a hint here as to when you use it over `lang_standardise()`?  Is it worth documenting them together?